src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -8593,6 +8596,7 @@ bool Unit::HandleProcTriggerSpell(Unit* victim, uint32 damage, AuraEffect* trigg
 
             target = this;
             trigger_spell_id = 22588;
+            [[fallthrough]]; // TODO: Not sure whether the fallthrough was a mistake (forgetting a break) or intended. This should be double-checked.
```

Comment:
@FrancescoBorzi  forgetting break proofs
https://github.com/SkittlesAreFalling/WoW-Carnage/blob/45d5d483c857bd21135f70e4abf278175ddf40e6/src/game/UnitAuraProcHandler.cpp#L905
https://github.com/Kittnz/ZeroServer/blob/193794dcdbda45a5bd512e6d83780e42308734e5/src/game/UnitAuraProcHandler.cpp#L905
https://github.com/IldebrandoS/brandonie-craft/blob/d36bf42f4ed163c821591c9d94539bd77d8462a2/game/UnitAuraProcHandler.cpp#L1236

Reply:
thanks @Viste we can open another PR for that!

Reply:
need break